recipes/cartopy_offlinedata/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cartopy_offlinedata" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/willirath/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: e7f3a6d47d97e133c516c631c1dbb366300c43f0dcae0d5c49c8a2a5810c477e
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - cartopy
+    - bash
```

Comment:
This should be in `build`.

@scopatz, I don't get why we need this. `conda-build` will call the `build.sh` script as `/bin/bash -e build.sh`, so there's bash already.

Reply:
@isuruf, thanks for the feedback.  I think @scopatz was originally thinking that we were calling a bash script that was part of the recipe rather than part of the installed package.  Given that's not the case, I don't think we need bash at all, so I took it out again.

@isuruf and @scopatz, any other changes you would like to see?

Reply:
Well, you are installing a bash script.  To make sure that it is runnable, bash should be a runtime dep

Reply:
Except that that script has `!#/bin/bash` as the shebang so it won't run if bash isn't installed there either way.  I'll do a PR upstream to change the shebang if that's what is holding up this PR getting merged.

Reply:
I also added `bash` as a build dependency.  That way, we're ready when the update happens upstream (see https://github.com/willirath/cartopy_offlinedata/pull/5)